### PR TITLE
add a service_period parameter for query subscription

### PR DESCRIPTION
**Pull request by mgates**

### CODE REVIEW

File: lib/conduit/reactor/actions/query_subscription.rb [hunk 1]
```diff
@@ -13,7 +14,7 @@ def remote_url
     private
 
     def query_subscription_url
-      "/#{@options[:mdn]}/query_subscription"
+      "/#{@options[:mdn]}/query_subscription?service_period_uuid=#{@options[:service_period_uuid]}"
```
Comment (pedromartinez) [inline lib/conduit/reactor/actions/query_subscription.rb]: Should you only append it if it's present?

File: lib/conduit/reactor/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Conduit
   module Reactor
-    VERSION = '0.0.39'
+    VERSION = '1.0.0'
```
Comment (pedromartinez) [inline lib/conduit/reactor/version.rb]: Yay!

File: lib/conduit/reactor/actions/query_subscription.rb (hunk 1 above)
Comment (mgates) [inline lib/conduit/reactor/actions/query_subscription.rb]: Well, maybe - https://github.com/Hello-Labs/reactor_api/pull/352 does the right think if it's nil.

Comment (pedromartinez) [inline lib/conduit/reactor/actions/query_subscription.rb]: Okay.  It just looks ugly.

**Comment by pedromartinez** [comment]
:+1:

**Comment by mgates** [comment]
Is this better? I'm pretty meh on it.

**Comment by pedromartinez** [comment]
It's fine.

**Pull request merged**